lib/profiles/modelsim.coffee

```diff
@@ -0,0 +1,45 @@
+Profile = require './profile'
+XRegExp = require('xregexp').XRegExp
+
+module.exports =
+  class Modelsim extends Profile
+    @profile_name: 'Modelsim'
+
+    scopes: ['source.vhdl' , 'source.verilog']
+
+    regex_string: '
+    (?<type> Error|Warning):[ ](?<file> [\\S]+\\.(?extensions))\\((?<row> [\\d]+)\\):[ ](?<message> .+)$
+    '
+
+    file_string: '
+    (?<file> [\\S]+\\.(?extensions))\\((?<row> [\\d]+)\\)
+    '
+
+    constructor: (output) ->
+      super(output)
+      @regex = @createRegex @regex_string
+      @regex_file = @createRegex @file_string
+
+    files: (line) ->
+      start = 0
+      out = []
+      while (m = XRegExp.exec line.substr(start), @regex_file)?
+        start += m.index
+        m.start = start
+        m.end = start + m.file.length + m.row.length + 1
+        m.col = '0'
+        start = m.end + 1
+        out.push m
+      out
+
+    in: (line) ->
+      if (m = XRegExp.exec line, @regex)? #Start of error message
+        m.col = 0
```

Comment:
There isn't but if `m.col` is set to `0` then AFAIK my implementation will not underline the error correctly. Could be mistaken though, I have to admit highlighting profiles are still a little messy.